spdx/file.py

```diff
@@ -26,9 +27,19 @@ class FileType(object):
     AUDIO = 6
     IMAGE = 7
     TEXT = 8
-    VIDEO = 9
     DOCUMENTATION = 9
     SPDX = 10
+    VIDEO = 11
+
+
+def file_type_from_rdf(rdf_file_type: str) -> str:
```

Comment:
Initially I thought that more places would require the actual string, but it is only used in one method. I moved the logic to this method instead so that it can return a `FileType`.